python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -654,6 +660,16 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 {%         if router_bgp.address_family_evpn.neighbor_default.next_hop_self_received_evpn_routes.inter_domain is arista.avd.defined(true) %}
 | L3 Gateway Inter-domain | True |
 {%         endif %}
+{%         if router_bgp.address_family_evpn.evpn_ethernet_segment is arista.avd.defined %}
```

Comment:
This check is built in to natural_sort, so can be skipped.

Reply:
Removed